pkg/image/prune/prune.go

```diff
@@ -39,8 +39,12 @@ const (
 	// not keep an ImageNode from being a candidate for pruning.
 	WeakReferencedImageEdgeKind = "WeakReferencedImage"
 
+	// ReferencedImageConfigEdgeKind defines an edge from an ImageStreamNode or an
+	// ImageNode to an ImageComponentNode.
+	ReferencedImageConfigEdgeKind = "ReferencedImageConfig"
+
 	// ReferencedImageLayerEdgeKind defines an edge from an ImageStreamNode or an
-	// ImageNode to an ImageLayerNode.
+	// ImageNode to an ImageComponentNode.
 	ReferencedImageLayerEdgeKind = "ReferencedImageLayer"
```

Comment:
Actually on second thought, you renamed the `ImageLayerNode` in the comment unnecessarily.